src/components/new-request/search.vue

```diff
@@ -38,18 +38,21 @@
       <v-col v-if="showBusinessLookup" cols="12" md="6" class="business-lookup">
         <template v-if="!business">
           <BusinessLookup
-            v-if="getIsAuthenticated"
+            v-if="isAuthenticated"
             :searchStatus="lookupActiveOrHistorical"
             @business="onBusiness($event)"
           />
-          <BusinessFetch v-else @business="onBusiness($event)"/>
+          <BusinessFetch
+            v-else
+            @business="onBusiness($event)"
+          />
         </template>
         <v-text-field
           v-else
           append-icon="mdi-close"
           readonly
           filled
-          :label="business.identifier"
```

Comment:
I thought this was cool but UI team didn't want it, so changing back to static label >>>

Reply:
FWIW, I also liked this and helped a lot with testing of XPROs.

Reply:
I was told, "change it back and we'll see what user feedback we get."

Reply:
I see. Got it.